csv2sql.drush.inc

```diff
@@ -37,6 +38,13 @@ function csv2sql_drush_command() {
 function drush_csv2sql($csv_path) {
   $prefix = drush_get_option('prefix', '_raw');
 
+  $limit = drush_get_option('limit', FALSE);
+
+  if ($limit && !is_numeric($limit)) {
```

Comment:
this allows passing 1.5
See example from [RESTful](https://github.com/RESTful-Drupal/restful/blob/24fe14f03a40564ac419e117d939f8074766bc78/plugins/restful/RestfulBase.php#L945-L947)